utils/rpms/daos.spec

```
@@ -87,6 +87,7 @@ BuildRequires: protobuf-c-devel
 BuildRequires: lz4-devel
 %endif
 BuildRequires: spdk-devel >= 21.07
+BuildRequires: spdk-devel < 22
```

While we typically want each dependency on it's own line, a version range can be specified on a single line:

```suggestion
BuildRequires: spdk-devel >= 21.07, spdk-devel < 22
```

That makes it more difficult to miss that it's actually a range, not just a _greater than or equal to_.